general/administration/Hardware-Acceleration/ubuntu-debian-nvenc.md

```diff
@@ -0,0 +1,161 @@
+
+   2. Scroll Down to the versions table or type in browser CTRL + F. Then type in your browser window for your drive
+        - IE. Geoforce GTX 970 = 432.21 (it showed us the driver on nvidias website)
+     
+   3. Once youve located your correct driver hover over the driver link with your mouse right click the link and copy the link location.
+
+   4. **(Important) YOU NEED TO RUN THESE COMMANDS IN CLI NOT IN YOUR DESKTOP TERMINAL** 
+        - To do this you need to go out of GUI(Desktop Enviroment) and into CLI(kernal recovery terminal). 
+        - So you do this by Pressing CTRL+ALT+F1 (Some people DE(desktop enviroment) will already be on F1 so change it to any other F#
+
+   5. Now your should see a terminal login with your user
+        - IE. my user is `devilscoder`      
+        - so i login with user `devilscoder` and the password associated with it.
+
+   6. Now make the this directory and navigate to it by sending this command.
+   
+                sudo mkdir /opt/nvidia && cd /opt/nvidia
+
+   7. Now you need to download the correctly update driver from the command posted below
+       
+                sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/(driver firmware number)/NVIDIA-Linux-x86_(driver firmware number).run
+        
+        - IE if my drivers firmware revision is 335.21 that you found on the nvidia driver search on hte nvidia website. So mine would look like this
+        
+                sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/435.21/NVIDIA-Linux-x86_64-435.21.run`
+
+   8. Now you need to give them permissions.
+                `sudo chmod +x ./NVIDIA-Linux-x86_64-(driver firmware number).run`
+   9.Now you need to run the driver installer.
+
+                ./NVIDIA-Linux-x86_64-430.50.run
+
+   10. Make sure the new drivers is installed and recognized by typing in
+                `nvidia-smi`
+
+   11. Once the driver is installed you now need to install and patch the driver with the patch file downloaded from the github.
+
+        Navigate to your users Home Directory either by using        
+               
+               `cd ~/`
+                
+        - or
+                `cd /home/(your username)`
+                
+        - Now run
+        
+                `sudo git clone https://ipfs.io/ipns/Qmed4r8yrBP162WK1ybd1DJWhLUi4t6mGuBoB9fLtjxR7u nvidia-patch`
+                
+        - Finally run
+        
+                `bash ./patch.sh`
+    
+   12. Now it should display succesfully patched. Now that everythings done you can restart xServer or whatever you were using for DE (Desktop Enviroment)    
+        - To start your xserver run    
+                `startx`
+        - Your Desktop Enviroment should then display login and move your way to your jellyfin install in browser
+            
+###  Configuring Your Settings in Jellyfin
+   1. Open up your jellyfin Web UI by typing in one of the following domains.        
```

Comment:
I guess if the user has got so far to use this guide you can assume he can open the web UI